fauntlet/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "fauntlet"
+version = "0.1.0"
+edition = "2021"
+license = "MIT/Apache-2.0"
+description = "Testing tool for comparing Skrifa and FreeType."
+repository = "https://github.com/googlefonts/fontations"
+readme = "README.md"
+publish = false
+
+[dependencies]
+skrifa = { version = "0.12.0", path = "../skrifa" }
+freetype-rs = "0.32.0"
```

Comment:
Good catch. I meant to document the FT version but forgot. Looks like this is actually 2.12.1. I’ll rev it. 

FreeType is bundled with the freetype-sys crate, compiled on build and statically linked. We can control the version by forking that repo, updating the C code and using a patch section in Cargo.toml to retarget the dependency.  This would prevent us from publishing on crates.io but I don’t see that as a problem. 

I personally prefer the wrapper library over calling FR directly because it avoids a lot of unsafe and general noise in the code. You can see the difference in this PR for the few cases where the wrapper doesn’t expose required functionality.

In any case, I’m happy to find a solution to test against ToT but I’d prefer not to block this PR on that.

Reply:
> Looks like this is actually 2.12.1. I’ll rev it.

What do you mean by "I'll rev it"? Is that something you do in the CI configuration?

Ack re using the library, but yes, I'd be good to have control over that and keep it frequently updated. Distro FreeTypes are old and insecure. Ok with me not to block on that but do that as follow-up.

Reply:
> What do you mean by "I'll rev it"? Is that something you do in the CI configuration?

Yes, this was using an older version of freetype_rs which ships with an older version of FT. Incidentally, updating this is what led to the diff with lsb deltas. 

Just to clarify the structure: freetype_rs is the wrapper library which depends on low level bindings in freetype-sys which bundles a copy of a specific version of the FT C code. So we don’t use distro copies.

Reply:
Filed testing against FreeType ToT as #691.